src/components/Article/Page.tsx

```diff
@@ -0,0 +1,53 @@
+import { useParams } from 'react-router-dom';
+
+import { useFetch } from '../../hooks/useFetch';
+import { ICurrentArticle } from '../../interfaces/interfaces';
+import { Header } from './Header';
+import { Description } from './Description';
+import { Navbar } from './Navbar';
+import { Sidebar } from './Sidebar';
+
+export function Page() {
+  const { slug } = useParams<string>();
+  const { data } = useFetch<ICurrentArticle>('/articles', `${slug}`);
```

Comment:
Couldn't we just pass the slug like this? 
`const { data } = useFetch<ICurrentArticle>('/articles/`${slug}`');`